MekHQ/src/mekhq/campaign/report/HangarReport.java

```diff
@@ -42,1144 +42,2227 @@
+        DefaultMutableTreeNode sSupportNavalSmall = new DefaultMutableTreeNode();
+        sSupportNaval.add(sSupportNavalSmall);
+
+        DefaultMutableTreeNode sSupportSub = new DefaultMutableTreeNode();
+        sSupportVees.add(sSupportSub);
+        DefaultMutableTreeNode sSupportSubLarge = new DefaultMutableTreeNode();
+        sSupportSub.add(sSupportSubLarge);
+        DefaultMutableTreeNode sSupportSubMedium = new DefaultMutableTreeNode();
+        sSupportSub.add(sSupportSubMedium);
+        DefaultMutableTreeNode sSupportSubSmall = new DefaultMutableTreeNode();
+        sSupportSub.add(sSupportSubSmall);
+
+        DefaultMutableTreeNode sSupportHydrofoil = new DefaultMutableTreeNode();
+        sSupportVees.add(sSupportHydrofoil);
+        DefaultMutableTreeNode sSupportHydrofoilLarge = new DefaultMutableTreeNode();
+        sSupportHydrofoil.add(sSupportHydrofoilLarge);
+        DefaultMutableTreeNode sSupportHydrofoilMedium = new DefaultMutableTreeNode();
+        sSupportHydrofoil.add(sSupportHydrofoilMedium);
+        DefaultMutableTreeNode sSupportHydrofoilSmall = new DefaultMutableTreeNode();
+        sSupportHydrofoil.add(sSupportHydrofoilSmall);
+
+        DefaultMutableTreeNode sSupportSatellite = new DefaultMutableTreeNode();
+        sSupportVees.add(sSupportSatellite);
+        DefaultMutableTreeNode sSupportSatelliteLarge = new DefaultMutableTreeNode();
+        sSupportSatellite.add(sSupportSatelliteLarge);
+        DefaultMutableTreeNode sSupportSatelliteMedium = new DefaultMutableTreeNode();
+        sSupportSatellite.add(sSupportSatelliteMedium);
+        DefaultMutableTreeNode sSupportSatelliteSmall = new DefaultMutableTreeNode();
+        sSupportSatellite.add(sSupportSatelliteSmall);
+
+        DefaultMutableTreeNode sSupportRail = new DefaultMutableTreeNode();
+        sSupportVees.add(sSupportRail);
+        DefaultMutableTreeNode sSupportRailLarge = new DefaultMutableTreeNode();
+        sSupportRail.add(sSupportRailLarge);
+        DefaultMutableTreeNode sSupportRailMedium = new DefaultMutableTreeNode();
+        sSupportRail.add(sSupportRailMedium);
+        DefaultMutableTreeNode sSupportRailSmall = new DefaultMutableTreeNode();
+        sSupportRail.add(sSupportRailSmall);
+
+        DefaultMutableTreeNode sSupportMaglev = new DefaultMutableTreeNode();
+        sSupportVees.add(sSupportMaglev);
+        DefaultMutableTreeNode sSupportMaglevLarge = new DefaultMutableTreeNode();
+        sSupportMaglev.add(sSupportMaglevLarge);
+        DefaultMutableTreeNode sSupportMaglevMedium = new DefaultMutableTreeNode();
+        sSupportMaglev.add(sSupportMaglevMedium);
+        DefaultMutableTreeNode sSupportMaglevSmall = new DefaultMutableTreeNode();
+        sSupportMaglev.add(sSupportMaglevSmall);
+
+
+        // Omni Support Vees
+        DefaultMutableTreeNode oSupportVees = new DefaultMutableTreeNode();
+        supportVees.add(oSupportVees);
+
+        DefaultMutableTreeNode oSupportWh
```

Comment:
```suggestion
                // should have been caught by this will otherwise be selected for other unit types
```

Reply:
Welp, this one is embarrassing... I pride myself on spelling things correctly 😂 
It has been fixed, and not through github cause I don't trust how the numbering looks (probably just found a bug on Github :joy:)